crates/rolldown/src/stages/generate_stage/code_splitting.rs

```diff
@@ -22,14 +22,14 @@ impl<'a> GenerateStage<'a> {
     };
     let meta = &self.link_output.metas[module_id];
 
-    if !module.is_included {
-      return;
-    }
-
-    if module_to_bits[module_id].has_bit(entry_index) {
-      return;
+    // Here if the module only has re-export statements, the `is_included` is false at `module_side_effects: false`.
```

Comment:
Is this the cause of unexpectedly generated common chunks on bundling vue?

Reply:
No. It is a another panic bug.

Reply:
It's probably related/